public/app/plugins/datasource/influxdb/FlightSQL.ts

```diff
@@ -0,0 +1,103 @@
+import { DataSourceInstanceSettings, TimeRange } from '@grafana/data';
+import { CompletionItemKind, LanguageDefinition, TableIdentifier } from '@grafana/experimental';
+import { SqlDatasource } from 'app/features/plugins/sql/datasource/SqlDatasource';
+import { DB, SQLQuery } from 'app/features/plugins/sql/types';
+import { formatSQL } from 'app/features/plugins/sql/utils/formatSQL';
+
+import { mapFieldsToTypes } from '../mysql/fields';
+import { buildColumnQuery, buildTableQuery, showDatabases } from '../mysql/mySqlMetaQuery';
+import { getSqlCompletionProvider } from '../mysql/sqlCompletionProvider';
+import { quoteIdentifierIfNecessary, quoteLiteral, toRawSql } from '../mysql/sqlUtil';
+import { MySQLOptions } from '../mysql/types';
+
+export class FlightSQL extends SqlDatasource {
```

Comment:
@itsmylife The other SQL datasources implement all of this stuff internally instead of importing from mysql like I did here, looks like a lot of the abstraction required to pull this in already exists